docarray/typing/tensor/ndarray.py

```diff
@@ -202,11 +202,11 @@ def to_protobuf(self) -> 'NdArrayProto':
         return nd_proto
 
     @staticmethod
-    def get_comp_backend() -> Type['NumpyCompBackend']:
+    def get_comp_backend() -> 'NumpyCompBackend':
         """Return the computational backend of the tensor"""
         from docarray.computation.numpy_backend import NumpyCompBackend
 
-        return NumpyCompBackend
+        return NumpyCompBackend()
```

Comment:
So this is the only main difference here, right?

Reply:
Main difference are two folds:
* Use Instance instead of class in get_comp_backend (to fix the static method pb)
* The return type of the function in the computational backend is always np.ndarray or torch.Tensor. Therefore mypy cannot now that if you passed a TorchTensor (from us) the method will return a TorchTensor. This is a limitation but it allow us to remove the second TypeVar Generic in the Computational backend. This means in some case we need either to tell mypy we know the return type will be a TorchTensor by calling cast or TorchTensor._docarray_from_native()